compressed_image_transport/cfg/CompressedSubscriber.cfg

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env python
+
+PACKAGE='compressed_image_transport'
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+
+mode_enum = gen.enum( [gen.const("unchanged", str_t, "unchanged", "keep image encoding"),
```

Comment:
Another point, more important, I think whatever the way the const are represented, it should be the same in both cfg files. Right now, the Publisher cfg uses strings...